src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -111,6 +116,8 @@ public async Task<IActionResult> Applications(string applicationType)
             
             switch (application.ApplicationStatus)
             {
+                case ApplicationStatus.Submitted:
```

Comment:
Do you need to detect if an Application is Resubmitted? We have this concept in Assessor

Reply:
I don't think we have a concept of Resubmitted in our workflow. They can respond to feedback but any significant changes would be a new application.